srcpkgs/fpc-source/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'fpc-source'
+pkgname=fpc-source
+version=3.0.4
+revision=1
+noarch=yes
+wrksrc="fpc-$version"
+short_desc="Source code for the Freepascal compiler and libs"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-2.0-or-later, modified LGPL-2.0-or-later"
+homepage="https://www.freepascal.org/"
+distfiles="$SOURCEFORGE_SITE/freepascal/Source/$version/fpc-$version.source.tar.gz"
+checksum=69b3b7667b72b6759cf27226df5eb54112ce3515ff5efb79d95ac14bac742845
+nostrip=yes
```

Comment:
why nostrip ?

Reply:
Took long time to strip  in my tests (I don' t remember what exactly, this is repackaged source code). I cannot replicate it,will remove.